packages/advi/advi.2.0.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Active DVI Dune package!"
+description: """
+
+Active-DVI is a DVI previewer and presenter written in OCaml with some eye
+candy effects for presentation, support for interactive demonstrations, and
+embedding of arbitrary applications within the presentation.
+
+Active-DVI is well suited for use in combination with WhizzyTeX
+(http://cristal.inria.fr/whizzytex/)
+"""
+maintainer: ["Didier Rémy <Didier.Remy@inria.fr>"]
+authors: [
+  "Alexandre Miquel"
+  "Jun Furuse"
+  "Didier Rémy"
+  "Pierre Weis"
+  "Xavier Leroy"
+  "Roberto Di Cosmo"
+  "Didier Le Botlan"
+  "Alan Schmitt"
+]
+license: "LGPL-2.1-or-later"
+homepage: "http://advi.inria.fr"
+dev-repo: "git+https://github.com:diremy/advi.git"
+doc: "http://advi.inria.fr"
+bug-reports: "Didier.Remy@inria.fr"
+depends: [
+  "ocaml-base-compiler" {>= "4.11.1"}
+  "dune" {>= "2.5"}
+  "graphics" {>= "5.1.1"}
+  "camlimages" {>= "5.0.4"}
+  "conf-texlive"
+  "conf-which"
```

Comment:
This improved but still fails on archlinux due to missing ghostscript seems.
Probably we need to add  `conf-ghostscript` for that, @avsm do you have a suggestion?
```

#=== ERROR while compiling advi.2.0.0 =========================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.12.0 | pinned(https://github.com/diremy/advi/archive/advi-2.0.0.tar.gz)
# path                 ~/.opam/4.12/.opam-switch/build/advi.2.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make
# exit-code            2
# env-file             ~/.opam/log/advi-13-2bfeb4.env
# output-file          ~/.opam/log/advi-13-2bfeb4.out
### output ###
# dune build --root=. src/Makefile.config
#         bash src/configure.sh (exit 1)
# (cd _build/default/src && /usr/sbin/bash discover.sh configure.sh)
# which: no gs in (/home/opam/.opam/4.12/.opam-switch/build/advi.2.0.0/_build/install/default/bin:/home/opam/.opam/4.12/bin:/home/opam/.opam/plugins/bin:/home/opam/.opam/4.12/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
# warning: kpathsea: configuration file texmf.cnf not found in these directories: /usr/bin:/usr/bin/share/texmf-local/web2c:/usr/bin/share/texmf-dist/web2c:/usr/bin/share/texmf/web2c:/usr/bin/texmf-local/web2c:/usr/bin/texmf-dist/web2c:/usr/bin/texmf/web2c:/usr:/usr/share/texmf-local/web2c:/usr/share/texmf-dist/web2c:/usr/share/texmf/web2c:/usr/texmf-local/web2c:/usr/texmf-dist/web2c:/usr/texmf/web2c://texmf-local/web2c:/://share/texmf-local/web2c://share/texmf-dist/web2c://share/texmf/web2c://texmf-local/web2c://texmf-dist/web2c://texmf/web2c.
# kpsewhich command:
# arcticle.cls
# failed, check your setup!
# make: *** [Makefile:28: src/Makefile.config] Error 1
```